src/overlays/actors/ovl_Demo_Ec/z_demo_ec.c

```diff
@@ -91,53 +92,30 @@ static s16 sAnimationObjects[] = {
     OBJECT_EC, OBJECT_EC, OBJECT_EC, OBJECT_EC, OBJECT_EC, OBJECT_EC, OBJECT_GM, OBJECT_MA2,
 };
 
-// skeletons
-extern FlexSkeletonHeader D_06013B88;
-extern FlexSkeletonHeader D_0600B7B8;
-extern FlexSkeletonHeader D_06006C90;
-extern FlexSkeletonHeader D_060000F0;
-extern FlexSkeletonHeader D_06007150;
-extern FlexSkeletonHeader D_06007958;
-extern FlexSkeletonHeader D_06000330;
-extern FlexSkeletonHeader D_0600BFA8;
-extern FlexSkeletonHeader D_060086D0;
-extern FlexSkeletonHeader D_06007FB8;
-extern FlexSkeletonHeader D_06002530;
-extern FlexSkeletonHeader D_06004258;
-extern FlexSkeletonHeader D_06004658;
-extern FlexSkeletonHeader D_060085F8;
-extern FlexSkeletonHeader D_06004868;
-extern FlexSkeletonHeader D_0600FEF0;
-extern FlexSkeletonHeader D_06008D90;
-
-// animations
-extern AnimationHeader D_0600BD38;
-extern AnimationHeader D_060048F4;
-extern AnimationHeader D_0600196C;
-extern AnimationHeader D_06005670;
-extern AnimationHeader D_06005EA8;
-extern AnimationHeader D_0600164C;
-extern AnimationHeader D_0600164C;
-extern AnimationHeader D_0600513C;
-extern AnimationHeader D_06002254;
-extern AnimationHeader D_0600C918;
-extern AnimationHeader D_06006C40;
-extern AnimationHeader D_060095DC;
-extern AnimationHeader D_06009BB4;
-extern AnimationHeader D_0600A07C;
-extern AnimationHeader D_0600A474;
-extern AnimationHeader D_06000CD8;
-extern AnimationHeader D_0600AFE0;
-extern AnimationHeader D_0600AFE0;
-extern AnimationHeader D_06002FA0;
-extern AnimationHeader D_06003A98;
-extern AnimationHeader D_06002FA0;
+// skeletons (all from other objects)
+// clang-format off
```

Comment:
What's the reason to disable clang-format here? We generally avoid it unless necessary

Reply:
I had turned it off because it made a mess of the comments I had left.

Reply:
its cause that one line is probably too long. id try to rewrite it in a way that doesnt get messed up by the formatter
the comments also need spaces after the //, which the formatter will do

Reply:
think this is one of the only outstanding comments left, if we wanna try to get this in